testing/test-helper.js

```diff
@@ -244,18 +244,42 @@ export class BrowserController {
     });
   }
 
+  /**
+   * Helper to support providing already-selected elements to methods.
+   * @param {string|Element} selectorOrElement
+   * @return {Element}
+   */
+  querySelectorOrElement(selectorOrElement) {
+    return selectorOrElement && selectorOrElement.constructor === String
```

Comment:
This will break in cross-origin iframes, since the `String` constructor will be different in each context.

```suggestion
    return typeof selectorOrElement === 'string'
```

Reply:
Is there a preferred cross-origin iframe-friendly mechanism to assert string vs element? Or do you think I should unwind this set of helpers altogether?

Reply:
`typeof` works for this.

Reply:
Done